sale_order_type_whole_delivered_invoiceability/tests/test_sale_order_type_whole_delivered_invoiceability.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2023 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import Form
+
+from odoo.addons.sale_order_whole_delivered_invoiceability.tests import (
+    test_sale_order_whole_delivered_invoiceability,
+)
+
+
+class TestSaleOrderTypeWholeDeliveredInvoiceability(
+    test_sale_order_whole_delivered_invoiceability.TestSaleOrderWholeDeliveredInvoiceability
+):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.sale_order_type = cls.env["sale.order.type"].create(
+            [
+                {
+                    "name": "Test Normal Order",
+                    "whole_delivered_invoiceability": True,
+                },
+            ]
+        )
+        cls.order = cls.env["sale.order"].create(
```

Comment:
Why can't you reuse the existing order?

Reply:
That was the intention, it has remained when deleting the excess, sorry. Now it's ready